README.md

```diff
@@ -552,120 +569,153 @@ times a second. You can test the following properties:
 * `contentLength`
 * `actualDataSize`
 
-And you can get the maximum, minimum, and average values for each with the
+You can get the maximum, minimum, and average values for each with the
 `crawler.queue.max`, `crawler.queue.min`, and `crawler.queue.avg` functions
-respectively. Like so:
+respectively. Like the `crawler.queue.get` method, these methods are pseudo
+asynchronous to support different backing stores for the queue. That means they
+will provide both a return value and a callback.
 
 ```js
-console.log("The maximum request latency was %dms.", crawler.queue.max("requestLatency"));
-console.log("The minimum download time was %dms.", crawler.queue.min("downloadTime"));
-console.log("The average resource size received is %d bytes.", crawler.queue.avg("actualDataSize"));
+crawler.queue.max("requestLatency", function (max) {
+    console.log("The maximum request latency was %dms.", max);
+});
+crawler.queue.min("downloadTime", function (min) {
+    console.log("The minimum download time was %dms.", min);
+});
+crawler.queue.avg("actualDataSize", function (avg) {
+    console.log("The average resource size received is %d bytes.", avg);
+});
 ```
 
-You'll probably often need to determine how many items in the queue have a given
-status at any one time, and/or retrieve them. That's easy with
+You'll probably often need to determine how many queue items have a given status
+and/or retrieve them. That's easily done with the methods
 `crawler.queue.countWithStatus` and `crawler.queue.getWithStatus`.
 
-`crawler.queue.countWithStatus` returns the number of queued items with a given
+`crawler.queue.countWithStatus` provides the number of queued items with a given
 status, while `crawler.queue.getWithStatus` returns an array of the queue items
-themselves.
+themselves. Again, by default, these methods both return and accept callbacks.
```

Comment:
`return and accept callbacks`, should probably be just `take callbacks`. I kinda feel like erasing the existence of the synchronous methods since they're totally legacy and people shouldn't be using them. :)